helm/net-exporter-chart/templates/daemonset.yaml

```diff
@@ -0,0 +1,41 @@
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: net-exporter
+  namespace: monitoring
+  labels:
+    app: net-exporter
+spec:
+  template:
+    metadata:
+      labels:
+        app: net-exporter
+      annotations:
+        releasetime: {{ $.Release.Time }}
+    spec:
+      containers:
+      - name: net-exporter
+        image: quay.io/giantswarm/net-exporter:[[ .SHA ]]
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8000
+            scheme: HTTP
+          initialDelaySeconds: 5
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 8000
+            scheme: HTTP
+          initialDelaySeconds: 5
+        resources:
+          requests:
+            cpu: 20m
+            memory: 20Mi
+          limits:
+            cpu: 20m
+            memory: 20Mi
```

Comment:
I feel like that will bite use quickly. Why not 50 at least? Today 20 is like 0 :D

Reply:
In testing, this runs at a stable 13-14Mib, so I went with my 'half again on top rounded nicely" rule of thumb.

50Mib is three and a half times what we need :D

Reply:
This is still GC-ed language. I think it has chance to OOM when GC kicks in. Up to you.